crates/uv-cli/src/lib.rs

```diff
@@ -235,6 +235,14 @@ pub struct GlobalArgs {
     /// Change to the given directory prior to running the command.
     #[arg(global = true, long, hide = true)]
     pub directory: Option<PathBuf>,
+
+    /// Run the command within the given project.
+    ///
+    /// All `pyproject.toml`, `uv.toml`, and `.python-version` files will be discovered by walking
+    /// up the directory tree from the project root, while relative paths will be resolved relative
+    /// to the current working directory.
```

Comment:
```suggestion
    /// up the directory tree from the project root, while other command-line arguments will be resolved relative
    /// to the current working directory.
```

This might be more clear. Some users could take "relative paths" to mean something else, like paths in project config? I'm not sure about the exact best way to say it.